representing-data/activity-representing-data.md

```diff
@@ -10,23 +10,91 @@ Our goal is to practice going from word problem to data structure, and to catch
+| --------- | -------------------------- | ---------------- | ----------------------------------- |
+| January   | 1,099                      | nikki_long       | 3                                   |
+| February  | 1,402                      | batwing.ardala   | 5                                   |
+| March     | 1,398                      | josephj4         | 8                                   |
+| April     | 989                        | danalikesanimals | 15                                  |
+| May       | 972                        | nikki_long       | 3                                   |
+| June      | 1,092                      | wary.berry       | 2                                   |
+| July      | 1,115                      | stephie933       | 12                                  |
+| August    | 1,108                      | wary.berry       | 2                                   |
+| September | 1,100                      | alexarivera      | 1                                   |
+| October   | 1,142                      | alexarivera      | 1                                   |
+| November  | 1,136                      | alexarivera      | 1                                   |
+| December  | 1,025                      | alexarivera      | 1                                   |
+
+From any part of this data, make a meaningful list.
+
+### Question 2: Adapt to a Dictionary
+
+Consider this table of data, which represents a summary of the patient's last visit.
+
+| Form Field         | Patient Answer                                      |
+| ------------------ | --------------------------------------------------- |
+| Patient First Name | Nelly                                               |
+| Date Registered    | 06/01/2015                                          |
+| Mailing Address    | 4829 Hazelwood Ave., Apt. 712, Des Moines, IA 50309 |
+| Email              | nelly.the.jelly                                     |
+| Policy Number      | 4001337200                                          |
+| RBC                | 5.78                                                |
+| Hematocrit %       | 17.9                                                |
+| Test Summary       | Satisfactory                                        |
+
+From any part of this data, make a meaningful dictionary.
+
+### Question 3: Create a Table
+
+Consider this situation:
+
+Imagine there is a small online conference named "Cool Dev Conf." The organizers collect and analyze attendee info during each conference, so they can understand which years and programming were the most popular. Cool Dev Conf is held yearly. The conference has many different event types: long talks, short talks, panels, workshops, and affinity groups. After 3 years of running Cool Dev Conf, Cool Dev Conf started to provide different tiers of scholarship tickets in order to encourage new attendees.
```

Comment:
Consider avoiding the term 'programming' to in this context, since we are more likely to expect it to be used in the context of coding.  Best to just avoid the possibility of confusion.
```suggestion
Imagine there is a small online conference named "Cool Dev Conf." The organizers collect and analyze attendee info during each conference, so they can understand which years and conference events were the most popular. Cool Dev Conf is held yearly. The conference has many different event types: long talks, short talks, panels, workshops, and affinity groups. After 3 years of running Cool Dev Conf, Cool Dev Conf started to provide different tiers of scholarship tickets in order to encourage new attendees.
```